src/test/java/step1/ExpressionTest.java

```diff
@@ -0,0 +1,39 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+class ExpressionTest {
+
+    @DisplayName("of")
+    @Nested
+    class Of {
+        @DisplayName("null이 전달된 경우")
+        @Test
+        void return_empty_expression_when_receive_null() {
+            // when
+            final Expression expression = Expression.of(null);
```

Comment:
null 테스트시 다음 어노테이션을 이용해도 좋을 것 같아요 :)

```suggestion
        @NullSource
        @ParameterizedTest
        void return_empty_expression_when_receive_null(String empty) {
            // when
            final Expression expression = Expression.of(empty);
```

Reply:
이런 방법이 있었군요 ㅎㅎ 감사합니다~